doc/guide.md

```diff
@@ -2737,6 +2737,8 @@ Defaults to `localhost`
 system-generated UUID (in cluster mode), or `localhost` (in dev mode)
 * `storage`: Defines the storage back end, `postgres`, `mongo` or (the default)
 `inmemory`
+* `healthPort`: port for readiness and liveness checks. Default value is
+   0 (disabled). Provide non-zero value to enable.
```

Comment:
```suggestion
* `healthPort`: port for the GET /readiness and GET /liveness health checks. Use 0 to disable, this is the default. They return 204 if Okapi is ready/responsive and 500 otherwise.
```

Reply:
ok